game/scripts/vscripts/items/trumps_fists.lua

```diff
@@ -21,13 +21,29 @@ function modifier_item_trumps_fists_passive:IsPurgable()
   return false
 end
 
-function modifier_item_trumps_fists_passive:OnCreated()
+function modifier_item_trumps_fists_passive:OnCreated(kv)
   self.bonus_all_stats = self:GetAbility():GetSpecialValueFor( "bonus_all_stats" )
   self.bonus_damage = self:GetAbility():GetSpecialValueFor( "bonus_damage" )
   self.bonus_health = self:GetAbility():GetSpecialValueFor( "bonus_health" )
   self.bonus_mana = self:GetAbility():GetSpecialValueFor( "bonus_mana" )
-
   self.heal_prevent_duration = self:GetAbility():GetSpecialValueFor( "heal_prevent_duration" )
+
+  if IsServer() then
+    local unit = self:GetCaster()
+
+    if unit.hero_projectile == nil then
+      unit.hero_projectile = unit:GetRangedProjectileName()
+    end
+
+    self.item_projectile = "particles/items/trumps_fists/trumps_fists_projectile.vpcf"
```

Comment:
There's no reason to save this on self if you aren't going to reference it anywhere else.